core/src/main/java/com/linecorp/armeria/server/AbstractPathMapping.java

```diff
@@ -84,7 +84,7 @@ public String loggerName() {
         return "__UNKNOWN__";
     }
 
-    static String loggerName(String pathish) {
+    protected static String loggerName(String pathish) {
```

Comment:
Oh good point - in that case I'm thinking to just drop this PR, no need to add visibility to a method with a good chance of going away (in the meantime I'm defining a path mapping where I'll use something else). What do you think?